packages/drivers/local-driver/src/localCreateDocument.ts

```diff
@@ -0,0 +1,49 @@
+/*!
+ * Copyright (c) Microsoft Corporation and contributors. All rights reserved.
+ * Licensed under the MIT License.
+ */
+
+import {
+	ensureFluidResolvedUrl,
+	getDocAttributesFromProtocolSummary,
+	getQuorumValuesFromProtocolSummary,
+	isCombinedAppAndProtocolSummary,
+} from "@fluidframework/driver-utils";
+import { ISummaryTree } from "@fluidframework/protocol-definitions";
+import { LocalDeltaConnectionServer } from "@fluidframework/server-local-server";
+import { defaultHash } from "@fluidframework/server-services-client";
+
+export async function createDocument(
+	localDeltaConnectionServer,
+	resolvedUrl,
+	summary: ISummaryTree,
+) {
+	ensureFluidResolvedUrl(resolvedUrl);
```

Comment:
rather than calling this, it would be better to just explicitly take `IFluidResolveUrl`, this will lead to a compile error rather than a runtime error, which is better.

Reply:
you can remove ensureFluidResolvedUrl now

Reply:
this change moved the responsibility to get a IFluidResolveUrl, this allows them to handle that however they want. The benefit is the caller won't get a runtime error, they will get a compile time error if they do it wrong. Compile errors happen much earlier in the development cycle so are easier to fix.